CIP-????/README.md

```diff
@@ -0,0 +1,474 @@
+---
+CIP: ????
+Title: Structured Contracts
+Category: Ledger/Tools
```

Comment:
```suggestion
Category: Tools
```
This was settled in https://github.com/cardano-foundation/CIPs/pull/963#discussion_r1916096054 and fixed in an earlier commit that was overwritten by the most recent force-push.

Reply:
no dispute of the `Tools` categorisation yet but we will confirm at tomorrow's CIP meeting... committing here since this change didn't make it into https://github.com/cardano-foundation/CIPs/pull/963/commits/144c0181ac62c3a5fe8c350d4ba2f7c1e4a9056b.